tattoo.ts

```diff
@@ -58,8 +58,16 @@ const cli = cliArgs([
     multiple: true,
     alias: "r",
     description:
-        "Explicit repos to process. Specifying explicit repos will disable" +
+        "Explicit repos to load. Specifying explicit repos will disable" +
         "running on the implicit set of repos for the user."
+  }, {
+    name: "test-repo",
+    type: String,
+    defaultValue: [],
```

Comment:
Why a `defaultValue` of `[]` but a type of `String`? The loop at https://github.com/Polymer/tattoo/blob/firstDraftComments/tattoo.ts#L437 seems like it could be done in the `type` argument.